dist/images/ovnkube.sh

```diff
@@ -775,12 +775,20 @@ ovn-node () {
   wait_for_event process_ready ovn-controller
   sleep 1
 
+  OVN_ENCAP_IP=""
+  ovn_encap_ip=`ovs-vsctl --if-exists get Open_vSwitch . external_ids:ovn-encap-ip | tr -d '\"'`
+  if [[ $? == 0 && ${ovn_encap_ip} != "" ]]; then
+    OVN_ENCAP_IP=$(echo "--encap-ip="${ovn_encap_ip})
```

Comment:
stylistically it would be better to put quotes around `${ovn_encap_ip}`. It's particularly weird to put quotes around `--encap-ip` which definitely doesn't have any spaces in it but then not have them also around the variable...

Reply:
will do